### ISSUE TITLE
Ouput of "Read Blueprint Entity" not stable

### ISSUE THREAD

**Issue by Zaltais333** [justarandomgeek/factorio-conman#9]
Whenever I am using the "Read Blueprint Entity" option (Blueprint=7 & W=0), I get an unstable output on control 2. It seems the outputs signals are oscillating at high frequency and it makes difficult to work with them.

**Comment by justarandomgeek** [comment]
Most entities will have more than one frame of data, it outputs them sequentially in pulses. You'll need a circuit that can capture and store multiple frames in rapid succession.

**Comment by Zaltais333** [comment]
Yes, I end up creating a circuit to save the ouput. However, will it possible to have an extra input signal to decide which frame to ouput ?
